server/helpers/fileHelper.js

```diff
@@ -0,0 +1,34 @@
+// For file uploads
+const multer = require('multer');
+const path = require('path');
+
+// Mount multer middleware before the route handler with a max file size of 10MB
+const preview = multer({
+    limits: { fileSize: 10 * 1024 * 1024 }, // Set the limit to 10MB
+
+    // File Filter to control the types of files uploaded
+    fileFilter(req, file, cb) {
+        if (!file.originalname.match(/\.(jpg|jpeg|png)$/)) {
+            return cb('Please upload an image with a valid format (jpg, jpeg, or png).', false);
```

Comment:
does there need to be a response status code sent back here?

Reply:
No, the handling of errors is in another function called "handleUploadError". It is also passed into the createEvent method (check the routes file). It is different cuz multer is weird like that.